src/exporter/index.js

```diff
@@ -33,7 +33,7 @@ module.exports = (hash, ipldResolver, options) => {
       return pull.empty()
     }
     return pull(
-      ipldResolver.getStream(new CID(item.hash)),
+      ipldResolver._getStream(new CID(item.hash)),
```

Comment:
It follows https://github.com/ipfs/interface-ipfs-core/issues/81#issuecomment-277271941